### ISSUE TITLE
org.cactoos.text.Replaced uses StringBuffer instead of StringBuilder

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/cactoos#1410]
The class `org.cactoos.text.Replaced` uses `StringBuffer` instead of `StringBuilder` ([here](https://github.com/yegor256/cactoos/blob/3675019c59c64477e8b80a04a25f86d717d9ac4b/src/main/java/org/cactoos/text/Replaced.java#L88)). Let's change it because `StringBuilder` is more efficient.

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso), @victornoel[/z](https://www.0crat.com/u/victornoel) please, pay attention to this issue

**Comment by victornoel** [comment]
@fabriciofx good idea, let's replace all usage of StringBuilder with StringBuffer then, and banish its use with forbiddenapis in the pom.xml (there is already an example there).
Same with Vector, Hashtable and Stac, see https://rules.sonarsource.com/java/RSPEC-1149

**Comment by victornoel** [comment]
@0crat in

**Comment by fabriciofx** [comment]
@victornoel Yeah, good idea! But we need take care beacuse some classes use `StringBuffer` intentionally, as `InputOfTest` and `TextOfTest`.

**Comment by victornoel** [comment]
@fabriciofx good point, yes!

**Comment by victornoel** [comment]
@0crat in

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1410#issuecomment-691661687))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1410](https://github.com/yegor256/cactoos/issues/1410) is [in scope](http://www.zerocracy.com/policy.html#14) for 2min
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @fabriciofx[/z](https://www.0crat.com/u/fabriciofx): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231410%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1410#issuecomment-860063656))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1410](https://github.com/yegor256/cactoos/issues/1410) is [in scope](http://www.zerocracy.com/policy.html#14) for 9mon
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @fabriciofx[/z](https://www.0crat.com/u/fabriciofx): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231410%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by yegor256** [comment]
Done: `ggrep -rn StringBuffer src/main/java` returns nothing, `Replaced` and the rest use `StringBuilder` now. Closing as done.
